File: crates/enforcer-memory/tests/fixtures/memory/lang_fsharp/Widget.fs
namespace Widgets

open System.Text

type Animal(name: string) =
    member this.Name = name

type Widget(name: string) =
    inherit Animal(name)

    member this.Draw() =
        if name = "" then
            printfn "unnamed"
        helper name

let helper label =
    printfn "helper"

File: crates/enforcer-memory/tests/fixtures/memory/lang_elixir/widget.ex
defmodule Widget do
  alias Helper.Text
  require Logger

  def draw(name) when name != "" do
    Logger.info("drawing")
    Text.upcase(name)
  end

  def draw(_name) do
    "unnamed"
  end

  defp helper(label) do
    label
  end
end
